### DISCUSSION TITLE
what should netsync be used for

### DISCUSSION THREAD

**Discussion by deltanedas** [space-wizards/space-station-14#16245] [Technical]
so far ive just seen it used for SpriteComponent but what about IconComponent and more?
if client can determine the state why not set it to false for everything

**Comment by metalgearsloth** [comment]
Nothing anymore now that sprite is client-only.

**Reply by deltanedas** [reply]
oh right pro

**Reply by ElectroJr** [reply]
It does still have some uses for anything that gets animated. E.g., several point-light components still have netsync disabled. For example, the lanterns have an animated light radius that grows and shrinks, so those components have netsync disabled to avoid unnecessarily dirtying & resetting of the component every tick.
